src/plugins/core/cell.ts

```diff
@@ -96,7 +98,10 @@ export class CellPlugin extends CorePlugin<CoreState> implements CoreState {
   allowDispatch(cmd: CoreCommand): CommandResult | CommandResult[] {
     switch (cmd.type) {
       case "UPDATE_CELL":
-        return this.checkCellOutOfSheet(cmd);
+        return this.checkValidations(
+          cmd,
+          this.chainValidations(this.checkCellOutOfSheet, this.checkUselessUpdateCell)
```

Comment:
the chain validation is actually useless. You can remove it. (same for CLEAR_CELL); There's only one test to adapt
```suggestion
          this.checkCellOutOfSheet, this.checkUselessUpdateCell
```